dhis-2/dhis-api/src/main/java/org/hisp/dhis/notification/logging/NotificationValidationResult.java

```diff
@@ -27,19 +27,30 @@
  */
 package org.hisp.dhis.notification.logging;
 
-import lombok.Builder;
-import lombok.Data;
-import org.hisp.dhis.program.notification.ProgramNotificationTemplate;
+import lombok.Getter;
 
-/**
- * @author Zubair Asghar
- */
-@Data
-@Builder
 public class NotificationValidationResult {
-  ProgramNotificationTemplate template;
+  private final boolean needsToCreateLogEntry;
+  @Getter private final boolean valid;
```

Comment:
we can also use `@Getter` for `needsToCreateLogEntry`.

Reply:
It is not using lombok to avoid the method to be called `isNeedsToCreateLogEntry`